README.md

```diff
@@ -135,21 +135,46 @@ logging_inputs:
     type: basics
 ```
 
-3) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the local system and remote rsyslog and outputs into the local files.
+3) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the remote rsyslog and outputs into the per host base local files.
```

Comment:
did you mean "per host"?  not sure what "base" means in this context

Reply:
Yes, I wanted to convert this line into a plain English... :p
`/var/log/remote/%HOSTNAME%/%PROGRAMNAME:::secpath-replace%.log`
With this path, remote logs are stored in the local files.
```
ls /var/log/remote/msg/_hostname_
abrt-hook-ccpp.log journal.log         newtag.log         setroubleshoot.log
auditd.log               kernel.log          nm-dispatcher.log  systemd.log
testTag0.log           ...
```
Is "outputs into the local files per host" better?

Reply:
> Is "outputs into the local files per host" better?

Yes, works for me